### PR TITLE
:ghost: Add swagger spec change notification for QE

### PR DESCRIPTION
**Pull request by aufi**
Related to https://issues.redhat.com/browse/MTA-615

### CODE REVIEW

File: .github/workflows/swagger-change-notification.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    name: Notify QE on Swagger spec file change
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+          :mega: There is a Swagger OpenAPI spec files change, please consider re-generate your API client. @mguetta1 @sshveta @kpunwatk
```
Comment (jortel) [inline .github/workflows/swagger-change-notification.yml:19]: @djzager Do you think we should we be hard coding user names in github actions?

Comment (aufi) [reply .github/workflows/swagger-change-notification.yml:19]: :+1: Would you mind create such team, it looks I don't have permissions to do that.

Comment (djzager) [reply .github/workflows/swagger-change-notification.yml:19]: Looks like we already have @konveyor/tackle-qe

**Review by djzager — APPROVED** [review]
lgtm

**Pull request merged**